hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetContentsHandler.java

```diff
@@ -105,6 +105,7 @@ public void validate(@NonNull final QueryContext context) throws PreCheckExcepti
      */
     private @Nullable Optional<FileContents> contentFile(
             @NonNull final FileID fileID, @NonNull final ReadableFileStore fileStore) {
+        requireNonNull(fileID);
```

Comment:
This method is private, so this line is unnecessary.  It won't be called with a null fileID.

Reply:
fixed thanks